packages/istanbul-reports/lib/html/annotator.js

```diff
@@ -142,6 +142,19 @@ function annotateBranches(fileCoverage, structuredText) {
         // only highlight if partial branches are missing or if there is a
         // single uncovered branch.
         if (sumCount > 0 || (sumCount === 0 && branchArray.length === 1)) {
+            // Need to recover the metaArray placeholder item to count an implicit else
+            if (
+                branchMeta[branchName].type === 'if' &&
+                branchArray.length === 2 &&
```

Comment:
I think adding a comment for what each of these lines reqpresents would be useful, does `branchArray.length === 2` imply that there's no `else?

Reply:
@bcoe
Will do Monday :)

Yes, the branchArray implies there's only the beginning if and the placeholder in the array.